### PR TITLE
Format node bindings with clang-format

### PR DESCRIPTION
**Pull request by matthewkeil**
No code changes were made to `kzg.cxx` only format was run

## Inclusions
- copy `.clang-format` to node bindings so IDE format command works
- run format on `kzg.cxx`
- add clang-format to `Makefile`

### CODE REVIEW

**Comment by matthewkeil** [comment]
@jtraglia I added the lint command to the `Makefile` but `clang-format` is not on the workflow VM.  What do you think is the best course of action for that?

File: bindings/node.js/src/kzg.cxx [hunk 1]
```diff
@@ -376,173 +388,201 @@ Napi::Value VerifyKzgProof(const Napi::CallbackInfo& info) {
...
  *
  * @remark blobs[0] relates to commitmentBytes[0] and proofBytes[0]
  *
  * @param[in] {Blob}    blobs - An array of serialized blobs to verify
- * @param[in] {Bytes48} commitmentBytes - An array of serialized commitments to verify
- * @param[in] {Bytes48} proofBytes - An array of serialized KZG proofs for verification
+ * @param[in] {Bytes48} commitmentBytes - An array of serialized commitments to
+ *                                        verify
+ * @param[in] {Bytes48} proofBytes - An array of serialized KZG proofs for
+ *                                   verification
```
Comment (matthewkeil) [inline bindings/node.js/src/kzg.cxx:435]: Not a fan of how these turned out.  They don't stack but I tried a couple options and this looks the least bad...  Open to suggestions @jtraglia

Comment (jtraglia) [reply bindings/node.js/src/kzg.cxx:435]: Yeah, I don't think it looks great either but I also think it's the best option.

**Review by jtraglia — APPROVED** [review]
LGTM 👍 If you guys want to change the formatting config later, that's fine too. I don't really care what the style is as long as everything is consistent.

**Pull request merged**